### PR TITLE
README: Add minor corrections and details

### PR DESCRIPTION
**Pull request by vchong**
Signed-off-by: Victor Chong victor.chong@linaro.org

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,16 +97,18 @@ $ portmap &
...
 $ /usr/bin/chromium/chrome --no-sandbox  \
-    --use-gl=egl --ozone-platform=wayland --no-sandbox --composite-to-mailbox --in-process-gpu --enable-low-end-device-mode \
+    --use-gl=egl --ozone-platform=wayland --no-sandbox --composite-to-mailbox \
+    --in-process-gpu --enable-low-end-device-mode \
     --enable-logging --v=0 \
     --start-maximized \
     --user-data-dir=data_dir \
     --blink-platform-log-channels=Media\
-    --register-pepper-plugins="/usr/lib/chromium/libopencdmadapter.so#ClearKey CDM#ClearKey CDM0.1.0.0#0.1.0.0;application/x-ppapi-open-cdm" \
+    --register-pepper-plugins="/usr/lib/chromium/libopencdmadapter.so#ClearKey \
```
Comment (zoltan-ongithub) [inline README.md]: Have you tested this command? It adds additional spaces to the input string passed to the register-pepper-plugins argument.

Comment (vchong) [inline README.md]: I ran it from a script and the browser launched. There were warnings and errors all over the place as I navigate to play the video, but it played without any issues, including pausing and playing again.

Comment (vchong) [inline README.md]: I can revert back to the original command if necessary, if you're more comfortable with that.

Comment (zoltan-ongithub) [inline README.md]: Could you revert this line? Everything else looks fine.

Comment (vchong) [inline README.md]: Updated.

**Comment by vchong** [comment]
Updated to select 'External ClearKey' instead of just 'ClearKey'.

**Comment by zoltan-ongithub** [comment]
Where is the update on "External Clearkey". It seems to be already added in 606ff647661bf7fe835d243b4089cb95a945342a.

**Comment by vchong** [comment]
Previously (in this PR) I changed 'External Clearkey' to just 'Clearkey', but that's wrong, so I updated the PR to change 'Clearkey' back to 'External Clearkey'. Sorry for the confusion.

**Pull request merged**